ui/pages/swaps/main-quote-summary/main-quote-summary.js

```diff
@@ -93,7 +93,6 @@ export default function MainQuoteSummary({
               position="bottom"
               html={amountToDisplay}
               disabled={ellipsedAmountToDisplay === amountToDisplay}
-              theme="white"
             >
               <span
                 className="main-quote-summary__quote-large-number"
```

Comment:
Removing other unneeded theme props in favour of default theme: `tippy-tooltip--mm-custom-theme`